heapSort.cpp

```diff
@@ -0,0 +1,75 @@
+#include<iostream>
+#include<stdlib.h>
+#include<process.h>
+#include<time.h>
```

Comment:
This all can be replaced by `#include <bits/stdc++.h>`

Reply:
@s-ayush2903 I have made the changes.